Configurations/windows-makefile.tmpl

```diff
@@ -460,10 +465,13 @@ $target: $deps "$ordinalsfile" "$mkdef_pl"
 	DEL $shlib.def.tmp
 	"\$(PERL)" "$mkrc_pl" $shlib$shlibext > $shlib.rc
 	\$(RC) \$(RCOUTFLAG)$shlib.res $shlib.rc
+	IF EXIST $shlib$shlibext.manifest DEL /F /Q $shlib$shlibext.manifest
 	\$(LD) \$(LDFLAGS) \$(LIB_LDFLAGS) \\
 		/implib:\$@ \$(LDOUTFLAG)$shlib$shlibext /def:$shlib.def @<< || (DEL /Q \$(\@B).* $shlib.* && EXIT 1)
 $objs $shlib.res$linklibs \$(EX_LIBS)
 <<
+	IF EXIST $shlib$shlibext.manifest \\
+	   \$(MT) \$(MTFLAGS) \$(MTINFLAG) $shlib$shlibext.manifest \$(MTOUTFLAG)$shlib$shlibext
```

Comment:
Regarding the use of $(MTINFLAG), you're entirely correct, it was intended to be used snuggly against its argument... But you know what nmake does with spaces at the end of macro values, right? 

There is a trick, and while I should have done it in that PR, I forgot. Will fix.